src/prometheus_alert_rules/cpu.rules

```diff
@@ -0,0 +1,27 @@
+groups:
+- name: HostCPU
+  rules:
+
+  # Compute number of cores
+  - record: node_cpu_cores_total
+    expr: count(node_cpu_seconds_total{mode="idle"}) without (cpu,mode)
+
+  # Alert for cpu load average per core on 5 minute interval: warning
+  - alert: HostCpuHighLoadAvg
+    expr: node_load1 / node_cpu_cores_total >= 8 * node_procs_running OR node_load5 / node_cpu_cores_total >= 4 * node_procs_running OR node_load15 / node_cpu_cores_total >= 2 * node_procs_running
+    for: 3m
+    labels:
+      severity: critical
```

Comment:
1. Perhaps there is a more direct (or indirect?) way to calculate high load, i.e. without needing to separately calculate number of cores?
2. Threshold are application specific, so should the severity here be `info`?